src/app/qt/facefilter/FrameHandler.cpp

```diff
@@ -176,11 +185,40 @@ cv::Size FrameHandlerManager::getSize() const
 }
 
 FrameHandlerManager*
-FrameHandlerManager::get(nlohmann::json* settings, const std::string& name, const std::string& description)
+FrameHandlerManager::get(const std::string& name, const std::string& description, const GLVersion &glVersion)
 {
     if (!m_instance)
     {
-        m_instance = new FrameHandlerManager(settings, name, description);
+        m_instance = new FrameHandlerManager(name, description, glVersion);
     }
     return m_instance;
 }
+
+//////
+
+std::unique_ptr<nlohmann::json> loadJSON(spdlog::logger& logger)
+{
+    std::unique_ptr<nlohmann::json> json;
+
+    QString inputFilename(":/facefilter.json");
+    QFile inputFile(inputFilename);
+    if (!inputFile.open(QIODevice::ReadOnly | QIODevice::Text))
+    {
+        logger.error("Can't open file");
+        return json;
+    }
+
+    QTextStream in(&inputFile);
+    //if (in.good())
```

Comment:
`if(in.status() != QTextStream::Ok)`.  This one is fairly catastrophic.   Will add an exception here.